src/IOUService/web/ymmd.js

```diff
@@ -38,10 +40,35 @@ function load_sent_notes() {
 }
 
 function show_sent_notes(notes) {
-    clear_table();
 
-    var table = document.getElementById("sent_notes_table");
-    $.each(notes, function(index, value) {
+    display_notes_table("sent_notes_table", notes);
+}
+
+function set_up_refresh_button() {
+    $("#refresh_button").click(function () {
+        load_received_notes();
+    });
+}
+
+function load_received_notes() {
+    $.ajax({
+        url: "/api/IOU/GetMyNotes",
+        type: "GET",
+        success: function (result) {
+            show_received_notes(result);
+        }
+    });
+}
+
+function show_received_notes(notes) {
+    display_notes_table("received_notes_table", notes);
+}
+
+function display_notes_table(identifier, notes) {
```

Comment:
Parameter name: identifier. This parameter is the id of the table to be populated. The name should reflect this